opencog/nlp/chatbot-psi/chatbot.scm

```diff
@@ -18,51 +18,62 @@
     (cancel-all-threads)
     (reset-all-states)
 
-    (let* ((sent-node (car (nlp-parse utterance)))
-           (list-of-words (get-word-list sent-node)))
-
-        (State input-utterance
-            (Reference
-                list-of-words
-                sent-node
-                (Node utterance)
-            )
+    (catch #t
```

Comment:
well, I understand that, but where is nlp-parse throwing an exception if it fails to parse? Most exceptions are thrown for programmer errors, and indicate a bug in the code, rather than invalid input data.